api/sheets/get_property.md

```diff
@@ -6,41 +6,55 @@ parent: Google Sheets
 grand_parent: API
 ---
 
-# Get the values of a given property for a node.
+# Retrieve property values for nodes
 
-## `=DCPROPERTY(dcids, property)`
+Given a list of nodes and a property label, returns values associated with the given property for each node.
 
-Given a property, together with a single DCID, a row of DCIDs, or a column of DCIDs, get the values of the given property for those DCIDs.
+## General information about this endpoint
 
-**Arguments**
-*    `dcids` - DCIDs to get the property for
-*    `property` - property to get
+**Signature**: `=DCPROPERTY(dcids, property)`
 
-**Returns**
+**Required arguments**:
 
-The properties of the DCIDs. For a single DCID, the result is a column of the given property's values for that DCID. For a row of DCIDs, the result is a matrix where each column contains the values of the given property for the DCID at the column's index. For a column of DCIDs, the result is a matrix where each row contains the values of the given property for the DCID at the row's index.
+*   `dcids`: A list of nodes to query, identified by their Data Commons identifiers.
+*   `property`: The property to query for.
 
-## Examples
+## Assembling the information you will need for a call to this method
 
-In this example, we find the `containedInPlace` of the Data Commons nodes for [California](https://datacommons.org/browser/geoId/06) and [Mountain View](https://datacommons.org/browser/geoId/0649670). The function call in the image uses a single call to get the names of both nodes, but it is equivalent to typing:
+Going into more detail on how to assemble the values for the required arguments:
 
-```
-=TRANSPOSE(DCPROPERTY("geoId/06", "containedInPlace"))
-```
+ - `dcids`: Data Commons uniquely identifies nodes by assigning them DCIDs, or Data Commons IDs. Your query will need to specify the DCIDs for the nodes of interest. More information about DCIDs is available in [the glossary](/glossary.html).
```

Comment:
Nice - can we keep the link to the glossary to the other sheet API pages?

Reply:
I don't understand?

Reply:
I think I meant that this sentence is a nice clear link to the glossary:
"More information about DCIDs is available in [the glossary](/glossary.html)."

and wondering if we should add that to the other sheets API pages as well

Reply:
Yep, we can (and will) incorporate this in other places!

Reply:
great thanks!